scripts/apps/authoring/authoring/components/CharacterCountConfigButton.tsx

```diff
@@ -15,7 +15,7 @@ export interface ICharacterLimitUiBehavior {
 export const DEFAULT_UI_FOR_EDITOR_LIMIT: CharacterLimitUiBehavior = 'highlight';
 
 interface IProps {
-    field: string;
+    field: string; // field ID
```

Comment:
Can we change the prop name to `fieldId` instead of adding a comment?

Reply:
It's used in many places in angular, so automatic rename doesn't work. I don't think it's a significant issue to spend the time to manually go over it.